<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_rev">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>26</y>
       <width>60</width>
       <height>55</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>AC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_per">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_div">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>60</width>
       <height>55</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mul">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sub">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>180</y>
       <width>60</width>
       <height>55</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_0">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>220</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_eq">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>230</y>
       <width>180</width>
       <height>46</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_dot">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="result_show">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sin">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>60</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>sin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tan">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>60</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>tan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mod">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>60</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>mod</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cos">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>60</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>cos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sqrt">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>180</y>
       <width>60</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_atan">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>130</y>
       <width>60</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>atan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_asin">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>60</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>asin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_acos">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>60</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgb(245, 100, 90);
 color: white;
 border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x4: 0, y3: 0, x4: 0, y2: 1, stop: 0 #FF7832: 1#FF9739);
}</string>
     </property>
     <property name="text">
      <string>acos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_graf">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>GRAF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_lb">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>(</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rb">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ln">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>ln</string>
     </property>
    </widget>
    <widget class="QCustomPlot" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>0</y>
       <width>301</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="spin_xy_1">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>280</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="spin_xy_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>280</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="spin_result_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>280</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="spin_result_1">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>280</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_x">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>25</y>
       <width>60</width>
       <height>55</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>credit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>deposit</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="pic1">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>311</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/2.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
